### PR TITLE
Feature/periodic acceptance tests

### LINKED ISSUE #160: CI: Add smoke tests running hourly
Run acceptance tests hourly and report errors to Slack

### PR DESCRIPTION
**Pull request by ekampf**
Resolves #160 

## Changes
- Defined hourly cron for acceptance tests
- Send failure to slack (#proj-terraform)

### CODE REVIEW

File: .github/workflows/smoketests.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        go mod download
+
+    - name: Acceptance tests
+      timeout-minutes: 10
+      env:
+        TWINGATE_URL: ${{ secrets.TWINGATE_URL }}
+        TWINGATE_NETWORK: ${{ secrets.TWINGATE_NETWORK }}
+        TWINGATE_API_TOKEN: ${{ secrets.TWINGATE_API_TOKEN }}
+      run: |
+        make testacc
```
Comment (romankor) [inline .github/workflows/smoketests.yml:74]: lets run `make sweep` at the end of of each acc test run

**Review by romankor — COMMENTED** [review]
lets run `make sweep` at the end of each run

File: .github/workflows/smoketests.yml (hunk 1 above)
Comment (ekampf) [reply .github/workflows/smoketests.yml:74]: @romankor  You cant...   what if a feature branch is also running in parallel?

**Review by romankor — APPROVED** [review]
GTM

**Pull request merged**